src/modules/create.js

```diff
@@ -0,0 +1,39 @@
+import { name } from './name.js';
+
+const gameURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+
+export async function createGame() {
```

Comment:
* Kindly ensure arrow functions are used instead of the function keyword if you check [Javascript requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code) you would see that the use of ES6 is encouraged. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern. Take a minute and also [go through this article on why ES6 arrow functions are so awesome](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html). Then work on the feedback given 😄